packages/transaction-controller/src/utils/simulation.ts

```diff
@@ -291,33 +288,32 @@ async function getTokenBalanceChanges(
 ): Promise<SimulationTokenBalanceChange[]> {
   const balanceTxs = getTokenBalanceTransactions(request, events);
 
-  const balanceTransactions = [
+  const transactions = [
     ...balanceTxs.before.values(),
     request,
     ...balanceTxs.after.values(),
   ];
 
-  log('Generated balance transactions', balanceTransactions);
+  log('Generated balance transactions', transactions);
```

Comment:
Minor, but this is also no longer just logging the balance transactions.

Reply:
Done.